2.0.0-alpha.3

matteobruni opened the pull request.

matteobruni committed:
fix: fixed gravity with new speed algorithm

matteobruni committed:
build: added bundle webpack configuration for bigCircles preset, this will be added to all presets

dependabot[bot] committed:
build(deps-dev): bump @octokit/core from 3.2.4 to 3.2.5

Bumps [@octokit/core](https://github.com/octokit/core.js) from 3.2.4 to 3.2.5.
- [Release notes](https://github.com/octokit/core.js/releases)
- [Commits](https://github.com/octokit/core.js/compare/v3.2.4...v3.2.5)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1254 from matteobruni/dependabot/npm_and_yarn/dev/octokit/core-3.2.5

build(deps-dev): bump @octokit/core from 3.2.4 to 3.2.5

matteobruni committed:
build: migrated fire preset to new format

dependabot[bot] committed:
build(deps): bump svelte from 3.31.2 to 3.32.0

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.31.2 to 3.32.0.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.31.2...v3.32.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1255 from matteobruni/dependabot/npm_and_yarn/dev/svelte-3.32.0

build(deps): bump svelte from 3.31.2 to 3.32.0

matteobruni committed:
build: migrated fountain preset to new format

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.1.28 to 1.1.29

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.1.28 to 1.1.29.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/commits)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1258 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.1.29

build(deps-dev): bump svelte-check from 1.1.28 to 1.1.29

matteobruni committed:
fix: some changes to movement and gravity

matteobruni committed:
fix: updated fountain preset

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
feat: working on add split particles, the new particle velocity needs improvements

dependabot[bot] committed:
build(deps): bump preact from 10.5.11 to 10.5.12

Bumps [preact](https://github.com/preactjs/preact) from 10.5.11 to 10.5.12.
- [Release notes](https://github.com/preactjs/preact/releases)
- [Commits](https://github.com/preactjs/preact/compare/10.5.11...10.5.12)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1261 from matteobruni/dependabot/npm_and_yarn/dev/preact-10.5.12

build(deps): bump preact from 10.5.11 to 10.5.12

matteobruni committed:
fix: fixed split bounce

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
build: website deploy prepared

matteobruni committed:
build: website startup

matteobruni committed:
build: git renamed website folder

matteobruni committed:
build(website): updated gtag

code-factor committed:
[CodeFactor] Apply fixes

[ci skip] [skip ci]

matteobruni committed:
build(website): clear cache command

matteobruni committed:
Merge branch 'website' of https://github.com/matteobruni/tsparticles into website

matteobruni committed:
chore: changed urls from particles.matteobruni.it to particles.js.org

matteobruni committed:
Merge pull request #1262 from matteobruni/website

New Website

matteobruni committed:
Merge branch 'master' into dev

# Conflicts:
#	components/preact/package.json
#	demo/jquery/views/index.html
#	main/core/README.md
#	main/core/deploy.docs.js
#	main/core/package.json
#	plugins/emitters/package.json
#	presets/60fps/README.md
#	presets/60fps/webpack.config.js
#	presets/backgroundMask/README.md
#	presets/backgroundMask/webpack.config.js
#	presets/basic/webpack.config.js
#	presets/bouncing/README.md
#	presets/bouncing/webpack.config.js
#	presets/fontAwesome/package.json
#	presets/fontAwesome/webpack.config.js
#	presets/links/package.json
#	presets/seaAnemone/package.json
#	shapes/circle/webpack.config.js
#	shapes/star/package.json
#	yarn.lock

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.1.29 to 1.1.30

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.1.29 to 1.1.30.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/commits)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
chore: updated links to new website

ghost commented:
## DeepCode's analysis on [#cfd625](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b7288468917436ad3baf04425d8f88d515e9e35d/matteobruni/tsparticles/cfd625246f193ee23919ffeb86648f3c6d74bb05/pr/_/%2F/code/?utm_source=gh_review&c=0&w=1&i=5&) found:

- :warning: **1** warning, :information_source: **5** minor issues. :point_down:
- :heavy_check_mark: **3** issues were fixed.
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">The body of this case clause duplicates the body of this case clause. This may be caused by a copy-paste error. If this usage is intentional, consider using fall-through to remove code duplication. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/cfd625246f193ee23919ffeb86648f3c6d74bb05/website/js/demo.js#L61">demo.js:61</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b7288468917436ad3baf04425d8f88d515e9e35d/matteobruni/tsparticles/cfd625246f193ee23919ffeb86648f3c6d74bb05/pr/_/%2Fwebsite%2Fjs%2Fdemo.js/javascript%2Fdc_interfile_project%2FDuplicateCaseBody/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/cfd625246f193ee23919ffeb86648f3c6d74bb05/interactions/particles/parallax/src/Parallax.ts#L9">Parallax.ts:9</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/cfd625246f193ee23919ffeb86648f3c6d74bb05/interactions/particles/parallax/src/Parallax.ts#L39">Parallax.ts:39</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/cfd625246f193ee23919ffeb86648f3c6d74bb05/interactions/particles/parallax/src/Parallax.ts#L45">Parallax.ts:45</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b7288468917436ad3baf04425d8f88d515e9e35d/matteobruni/tsparticles/cfd625246f193ee23919ffeb86648f3c6d74bb05/pr/_/%2Finteractions%2Fparticles%2Fparallax%2Fsrc%2FParallax.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">Switch statement should include a 'default' case Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/cfd625246f193ee23919ffeb86648f3c6d74bb05/plugins/polygonMask/src/Utils.ts#L83">Utils.ts:83</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b7288468917436ad3baf04425d8f88d515e9e35d/matteobruni/tsparticles/cfd625246f193ee23919ffeb86648f3c6d74bb05/pr/_/%2Fplugins%2FpolygonMask%2Fsrc%2FUtils.ts/javascript%2Ftslint%2Fswitch-default/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b7288468917436ad3baf04425d8f88d515e9e35d/matteobruni/tsparticles/cfd625246f193ee23919ffeb86648f3c6d74bb05/pr/_/%2F/code/?utm_source=gh_review&c=0&w=1&i=5&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
Merge pull request #1263 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.1.30

build(deps-dev): bump svelte-check from 1.1.29 to 1.1.30

matteobruni committed:
Create deploy.site.yml

matteobruni committed:
build: added website to lerna

matteobruni committed:
build: fixed website deploy script

matteobruni committed:
Update deploy.site.yml

dependabot[bot] committed:
build(deps-dev): bump svelte-preprocess from 4.6.3 to 4.6.5

Bumps [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess) from 4.6.3 to 4.6.5.
- [Release notes](https://github.com/sveltejs/svelte-preprocess/releases)
- [Changelog](https://github.com/sveltejs/svelte-preprocess/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte-preprocess/compare/v4.6.3...v4.6.5)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1264 from matteobruni/matteobruni-patch-1

Create deploy.site.yml

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.1.30 to 1.1.31

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.1.30 to 1.1.31.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/commits/svelte-check-1.1.31)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1265 from matteobruni/dependabot/npm_and_yarn/dev/svelte-preprocess-4.6.5

build(deps-dev): bump svelte-preprocess from 4.6.3 to 4.6.5

matteobruni committed:
Merge pull request #1266 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.1.31

build(deps-dev): bump svelte-check from 1.1.30 to 1.1.31

matteobruni committed:
build: website updated

matteobruni committed:
build: website updated

matteobruni committed:
build: preparing 404 page

matteobruni committed:
build: completed 404 page

matteobruni committed:
build: completed 404 page

matteobruni committed:
chore(workflow): removed node.js 10.x from build

matteobruni committed:
chore(workflow): removed node.js 10.x from build

matteobruni committed:
removed node.js 10.x from CI

matteobruni committed:
Merge pull request #1270 from matteobruni/website-update

Website update

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-typescript from 8.1.0 to 8.1.1

Bumps [@rollup/plugin-typescript](https://github.com/rollup/plugins) from 8.1.0 to 8.1.1.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Commits](https://github.com/rollup/plugins/compare/typescript-v8.1.0...typescript-v8.1.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-commonjs from 17.0.0 to 17.1.0

Bumps [@rollup/plugin-commonjs](https://github.com/rollup/plugins) from 17.0.0 to 17.1.0.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Commits](https://github.com/rollup/plugins/compare/commonjs-v17.0.0...commonjs-v17.1.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump sirv-cli from 1.0.10 to 1.0.11

Bumps [sirv-cli](https://github.com/lukeed/sirv) from 1.0.10 to 1.0.11.
- [Release notes](https://github.com/lukeed/sirv/releases)
- [Commits](https://github.com/lukeed/sirv/compare/v1.0.10...v1.0.11)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1271 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-typescript-8.1.1

build(deps-dev): bump @rollup/plugin-typescript from 8.1.0 to 8.1.1

matteobruni committed:
Merge pull request #1272 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-commonjs-17.1.0

build(deps-dev): bump @rollup/plugin-commonjs from 17.0.0 to 17.1.0

matteobruni committed:
Merge pull request #1273 from matteobruni/dependabot/npm_and_yarn/dev/sirv-cli-1.0.11

build(deps): bump sirv-cli from 1.0.10 to 1.0.11

dependabot[bot] committed:
build(deps): bump svelte from 3.32.0 to 3.32.1

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.32.0 to 3.32.1.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.32.0...v3.32.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Update deploy.js

matteobruni committed:
Merge branch 'master' into website-update

matteobruni committed:
Merge pull request #1275 from matteobruni/website-update

Website Deploy Updated

matteobruni committed:
build: updated yarn lock

matteobruni committed:
Merge pull request #1274 from matteobruni/dependabot/npm_and_yarn/dev/svelte-3.32.1

build(deps): bump svelte from 3.32.0 to 3.32.1

matteobruni committed:
Merge branch 'master' into website-update

matteobruni committed:
Merge pull request #1276 from matteobruni/website-update

Website Deploy Updated

matteobruni committed:
Merge branch 'master' into dev

# Conflicts:
#	yarn.lock

matteobruni committed:
build: updates from master

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.12.5 to 3.12.6

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.12.5 to 3.12.6.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.12.5...v3.12.6)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1279 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.12.6

build(deps-dev): bump uglify-js from 3.12.5 to 3.12.6

matteobruni committed:
build: polygon mask plugin refactoring

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.1.31 to 1.1.32

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.1.31 to 1.1.32.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-1.1.31...svelte-check-1.1.32)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1280 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.1.32

build(deps-dev): bump svelte-check from 1.1.31 to 1.1.32

matteobruni committed:
build: updated polygon mask instance

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
Update nodejs.yml

dependabot[bot] committed:
build(deps): bump minify from 6.0.1 to 7.0.0

Bumps [minify](https://github.com/coderaiser/minify) from 6.0.1 to 7.0.0.
- [Release notes](https://github.com/coderaiser/minify/releases)
- [Changelog](https://github.com/coderaiser/minify/blob/master/ChangeLog)
- [Commits](https://github.com/coderaiser/minify/compare/v6.0.1...v7.0.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1281 from matteobruni/dependabot/npm_and_yarn/dev/minify-7.0.0

build(deps): bump minify from 6.0.1 to 7.0.0

dependabot[bot] committed:
build(deps-dev): bump ng-packagr from 11.1.2 to 11.1.3

Bumps [ng-packagr](https://github.com/ng-packagr/ng-packagr) from 11.1.2 to 11.1.3.
- [Release notes](https://github.com/ng-packagr/ng-packagr/releases)
- [Changelog](https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ng-packagr/ng-packagr/compare/v11.1.2...v11.1.3)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1282 from matteobruni/dependabot/npm_and_yarn/dev/ng-packagr-11.1.3

build(deps-dev): bump ng-packagr from 11.1.2 to 11.1.3

matteobruni committed:
build: fixed angular demo

matteobruni committed:
build: fixed vue2 and vue3 demos

matteobruni committed:
build: working on svelte demo

dependabot[bot] committed:
build(deps-dev): bump svelte-preprocess from 4.6.5 to 4.6.6

Bumps [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess) from 4.6.5 to 4.6.6.
- [Release notes](https://github.com/sveltejs/svelte-preprocess/releases)
- [Changelog](https://github.com/sveltejs/svelte-preprocess/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte-preprocess/compare/v4.6.5...v4.6.6)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1283 from matteobruni/dependabot/npm_and_yarn/dev/svelte-preprocess-4.6.6

build(deps-dev): bump svelte-preprocess from 4.6.5 to 4.6.6

dependabot[bot] committed:
build(deps-dev): bump chai from 4.2.0 to 4.3.0

Bumps [chai](https://github.com/chaijs/chai) from 4.2.0 to 4.3.0.
- [Release notes](https://github.com/chaijs/chai/releases)
- [Changelog](https://github.com/chaijs/chai/blob/master/History.md)
- [Commits](https://github.com/chaijs/chai/compare/4.2.0...4.3.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump ts-loader from 8.0.14 to 8.0.15

Bumps [ts-loader](https://github.com/TypeStrong/ts-loader) from 8.0.14 to 8.0.15.
- [Release notes](https://github.com/TypeStrong/ts-loader/releases)
- [Changelog](https://github.com/TypeStrong/ts-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/TypeStrong/ts-loader/compare/v8.0.14...v8.0.15)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1285 from matteobruni/dependabot/npm_and_yarn/dev/chai-4.3.0

build(deps-dev): bump chai from 4.2.0 to 4.3.0

matteobruni committed:
Merge pull request #1286 from matteobruni/dependabot/npm_and_yarn/dev/ts-loader-8.0.15

build(deps-dev): bump ts-loader from 8.0.14 to 8.0.15

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: some deps updated

matteobruni committed:
build: updated some deps

matteobruni committed:
build: updated some deps

matteobruni committed:
build: fixed vue externals

matteobruni committed:
chore: added move and parallax particles interactors

matteobruni committed:
build: fixed presets dependencies

matteobruni committed:
build: updated presets, fixed react demo build

matteobruni committed:
build: fixed all presets, created a demo page for presets

matteobruni committed:
build: fixed stars preset

matteobruni committed:
chore(release): published new version

 - ng-particles@3.0.0-alpha.3
 - inferno-particles@2.0.0-alpha.3
 - jquery-particles@2.0.0-alpha.3
 - preact-particles@2.0.0-alpha.3
 - react-tsparticles@2.0.0-alpha.3
 - svelte-particles@2.0.0-alpha.3
 - particles.vue@3.0.0-alpha.3
 - particles.vue3@2.0.0-alpha.3
 - ng-particles-demo@2.0.0-alpha.3
 - editor-demo@2.0.0-alpha.3
 - inferno-particles-demo@2.0.0-alpha.3
 - jquery-particles-demo@2.0.0-alpha.3
 - tsparticles-demo@2.0.0-alpha.3
 - preact-particles-demo@2.0.0-alpha.3
 - react-tsparticles-demo@2.0.0-alpha.3
 - svelte-demo@2.0.0-alpha.3
 - particles.vue-demo@2.0.0-alpha.3
 - particles.vue3-demo@2.0.0-alpha.3
 - tsparticles-editor@2.0.0-alpha.3
 - tsparticles-interaction-light@2.0.0-alpha.3
 - tsparticles-interaction-external-attract@2.0.0-alpha.3
 - tsparticles-interaction-external-bounce@2.0.0-alpha.3
 - tsparticles-interaction-external-bubble@2.0.0-alpha.3
 - tsparticles-interaction-external-connect@2.0.0-alpha.3
 - tsparticles-interaction-external-grab@2.0.0-alpha.3
 - tsparticles-interaction-external-repulse@2.0.0-alpha.3
 - tsparticles-interaction-external-trail@2.0.0-alpha.3
 - tsparticles-interaction-particles-attract@2.0.0-alpha.3
 - tsparticles-interaction-particles-collisions@2.0.0-alpha.3
 - tsparticles-interaction-particles-links@2.0.0-alpha.3
 - tsparticles-interaction-particles-move@2.0.0-alpha.3
 - tsparticles-interaction-particles-parallax@2.0.0-alpha.3
 - tsparticles-interaction-particles-repulse@2.0.0-alpha.3
 - tsparticles-core@2.0.0-alpha.3
 - tsparticles@2.0.0-alpha.3
 - tsparticles-slim@2.0.0-alpha.3
 - tsparticles-plugin-absorbers@2.0.0-alpha.3
 - tsparticles-plugin-emitters@2.0.0-alpha.3
 - tsparticles-plugin-infection@2.0.0-alpha.3
 - tsparticles-plugin-polygon-mask@2.0.0-alpha.3
 - tsparticles-preset-big-circles@2.0.0-alpha.3
 - tsparticles-preset-fire@2.0.0-alpha.3
 - tsparticles-preset-fountain@2.0.0-alpha.3
 - tsparticles-preset-links@2.0.0-alpha.3
 - tsparticles-preset-sea-anemone@2.0.0-alpha.3
 - tsparticles-preset-snow@2.0.0-alpha.3
 - tsparticles-preset-stars@2.0.0-alpha.3
 - tsparticles-shape-bubble@2.0.0-alpha.3
 - tsparticles-shape-circle@2.0.0-alpha.3
 - tsparticles-shape-heart@2.0.0-alpha.3
 - tsparticles-shape-image@2.0.0-alpha.3
 - tsparticles-shape-line@2.0.0-alpha.3
 - tsparticles-shape-multiline-text@2.0.0-alpha.3
 - tsparticles-shape-polygon@2.0.0-alpha.3
 - tsparticles-shape-spiral@2.0.0-alpha.3
 - tsparticles-shape-square@2.0.0-alpha.3
 - tsparticles-shape-star@2.0.0-alpha.3
 - tsparticles-shape-text@2.0.0-alpha.3
 - cra-template-particles-typescript@2.0.0-alpha.3
 - cra-template-particles@2.0.0-alpha.3
 - tsparticles-updater-angle@2.0.0-alpha.3
 - tsparticles-updater-color@2.0.0-alpha.3
 - tsparticles-updater-life@2.0.0-alpha.3
 - tsparticles-updater-opacity@2.0.0-alpha.3
 - tsparticles-updater-orbit@2.0.0-alpha.3
 - tsparticles-updater-out-modes@2.0.0-alpha.3
 - tsparticles-updater-size@2.0.0-alpha.3
 - tsparticles-updater-stroke-color@2.0.0-alpha.3
 - tsparticles-website@2.0.0-alpha.3

matteobruni committed:
build: version bump
